### PR TITLE
PARQUET-1527:  [parquet-tools] cat command throw java.lang.ClassCastException

### PR DESCRIPTION
**Pull request by masayuki038**
I download sources from release/1.11.0 and built.
https://github.com/apache/parquet-mr/releases/tag/apache-parquet-1.11.0

As I executed cat command of parquet-tools, the below exception occurred.
```
java -jar ./parquet-tools-1.11.0.jar cat ./v11.parquet
java.lang.ClassCastException: required int32 int32_field is not a group
```

### CODE REVIEW

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+       .append(BINARY_FIELD, Binary.fromString("foobar"))
+       .append(FIXED_LEN_BYTE_ARRAY_FIELD,
+         Binary.fromConstantByteArray(new byte[]{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 })));
+    } finally {
+      writer.close();
+    }
+  }
+
+  private File createTempFile() throws IOException {
+    File tmp = File.createTempFile(getClass().getSimpleName(), ".tmp");
```
Comment (Fokko) [inline parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: I would replace this one with a junit rule: https://garygregory.wordpress.com/2010/01/20/junit-tip-use-rules-to-manage-temporary-files-and-folders/

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    );
+  }
+
+  private void write(MessageType schema, File f) throws IOException {
+    Path fsPpath = new Path(f.getPath());
+    Configuration conf = new Configuration();
+    SimpleGroupFactory fact = new SimpleGroupFactory(schema);
+    GroupWriteSupport.setSchema(schema, conf);
+
+    ParquetWriter<Group> writer = new ParquetWriter<Group>(
```
Comment (Fokko) [inline parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: I would wrap an try-with-resource around the writer.

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    MessageType schema = createSchema();
+    write(schema, testFile);
+  }
+
+  @After
+  public void tearDown() {
+    this.testFile.delete();
+  }
+
+  private MessageType createSchema() {
```
Comment (Fokko) [inline parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: We can make this static.

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  @Before
+  public void setUp() throws IOException {
+    this.testFile = createTempFile();
+    MessageType schema = createSchema();
+    write(schema, testFile);
+  }
+
+  @After
+  public void tearDown() {
+    this.testFile.delete();
```
Comment (Fokko) [inline parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: When using the junit [TemporaryFolder](https://garygregory.wordpress.com/2010/01/20/junit-tip-use-rules-to-manage-temporary-files-and-folders/) we don't need this anymore.

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  private static final String DOUBLE_FIELD = "double_field";
+  private static final String BINARY_FIELD = "binary_field";
+  private static final String FIXED_LEN_BYTE_ARRAY_FIELD = "flba_field";
+
+
+  private File testFile;
+
+  @Test
+  public void testConverter() throws IOException {
+    ParquetReader<SimpleRecord> reader =
```
Comment (Fokko) [inline parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: The reader is not being closed, I would use a try-with-resources here.

File: parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java (hunk 1 above)
Comment (masayuki038) [reply parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleRecordConverter.java]: @Fokko I did not know about Temporary Folder. Thanks !!

**Pull request merged**